src/kvstore/NebulaStore.h

```diff
@@ -264,6 +264,10 @@ class NebulaStore : public KVStore, public Handler {
                               PartitionID partId,
                               const std::vector<HostAddr>& remoteListeners) override;
 
+    raftex::RaftexService* getRaftexService() const {
```

Comment:
There is an method `part` in `NebulaStore`, you could just use it.

Reply:
good catch, fixed